### PR TITLE
Spectroscopic Data Reduction: trace, extract, calibrate & data

### PR DESCRIPTION
**Pull request by keflavich**
Tutorial 3 following #508, #510

### CODE REVIEW

**Comment by adrn** [comment]
I fixed the issues with the color-excess tutorial (causing your builds to fail), so if you rebase this and your other PRs (sorry...) the CI should start working again. Sorry for the headache!

**Comment by keflavich** [comment]
I've addressed @eblur's comments and applied some other changes that were needed (header, import location, etc)

File: tutorials/SpectroscopicDataReductionBasics/3-Trace_Extract_Wavelength-CalibrateSpectrum.ipynb [hunk 1]
```diff
@@ -0,0 +1,413 @@
+{
```
Comment (kelle) [inline tutorials/SpectroscopicDataReductionBasics/3-Trace_Extract_Wavelength-CalibrateSpectrum.ipynb:1]: <p>cut "Part" from the title.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/511/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (kelle) [inline tutorials/SpectroscopicDataReductionBasics/3-Trace_Extract_Wavelength-CalibrateSpectrum.ipynb:1]: <p>gone through Spectroscopic Data Reduction 1 and 2 on finding the trace and wavelength calibration.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/511/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (kelle) [inline tutorials/SpectroscopicDataReductionBasics/3-Trace_Extract_Wavelength-CalibrateSpectrum.ipynb:1]: <p>link to docs https://docs.astropy.org/en/stable/units/equivalencies.html#spectral-doppler-equivalencies</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/511/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Comment (kelle) [inline tutorials/SpectroscopicDataReductionBasics/3-Trace_Extract_Wavelength-CalibrateSpectrum.ipynb:1]: <p>Maybe open an issue on astropy-tutorials for a flux calibration tutorial and link to it here.</p><p>A tutorial on flux calibration is forthcoming (astropy-tutorials#570) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/511/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

**Review by kelle — CHANGES_REQUESTED** [review]
Teeny comments.

**Review by kelle — APPROVED** [review]

**Pull request merged**